platforms/software/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/BuildInitPluginIntegrationTest.groovy

```diff
@@ -293,6 +293,10 @@ class BuildInitPluginIntegrationTest extends AbstractInitIntegrationSpec {
 
         then:
         outputContains("""Options
+     --dialog     Opposite option of --no-dialog.
```

Comment:
This is very awkward to read: `Opposite option of --no-dialog.`

Reply:
This no longer applies after the option rename